gui/src/pages/WorkspaceInfo.svelte

```diff
@@ -0,0 +1,51 @@
+<script lang="ts">
+  import Layout from '../components/Layout.svelte';
+  import WorkspaceNav from '../components/WorkspaceNav.svelte';
+  import CenterFormPanel from '../components/form/CenterFormPanel.svelte';
+  import { api } from '../lib/api';
+  import Spinner from '../components/Spinner.svelte';
+
+  export let params = { workspace: '' };
+
+  const workspaceId = params.workspace;
+  const workspace = api.workspace(workspaceId);
+  const workspaceRoute = `/workspaces/${encodeURIComponent(workspaceId)}`;
+</script>
+
+<Layout>
+  <WorkspaceNav {workspaceId} active="Dashboard" slot="navbar" />
+  <CenterFormPanel title="Workspace details" backRoute={workspaceRoute}>
+    {#await workspace.describeSelf()}
+      <Spinner />
+    {:then description}
+      <table>
+        <tr>
+          <th>ID</th>
+          <td>{description.id}</td>
+        </tr>
+        <tr>
+          <th>Display name</th>
+          <td>{description.displayName}</td>
+        </tr>
+        <tr>
+          <th>Root directory</th>
+          <td>{description.root}</td>
+        </tr>
+      </table>
+    {/await}
+
+    <!-- Databases, Apps, cloud services, etc. -->
+  </CenterFormPanel>
+</Layout>
+
+<style>
```

Comment:
I would use `<CheckeredTableWrapper />` instead of doing this custom styling, and I would also recommend using the two-column `.label` and value approach used in the process details and environment pages currently instead of putting these `<th>`s above `<td>`s

Reply:
Whoops, didn't realize this already did two-column. Regardless, please conform this to the way we do it in the two pages mentioned above so that we don't have multiple visually different implementations of the same kind of table. I'll probably make a `KeyValueTable` component like we had in the old console (I think they were called "Descriptions" or "Summaries" per the old design framework?)